index.bs

```diff
@@ -305,18 +253,25 @@ spec: mimesniff; urlPrefix: https://mimesniff.spec.whatwg.org/#
 
       <p>
         The <dfn for='VideoConfiguration' dict-member>contentType</dfn> member
-        represents the MIME type of the video track.
+        represents the [=MIME type=] of the video track.
       </p>
 
       <p>
         To check if a {{VideoConfiguration}} <var>configuration</var> is a
         <dfn>valid video configuration</dfn>, the following steps MUST be run:
         <ol>
           <li>
-            If <var>configuration</var>'s {{VideoConfiguration/contentType}} is
-            not a <a>valid video MIME type</a>, return <code>false</code> and
+            Let |mimeType| be the result of running [=parse a MIME type=]
+            with |configuration|'s {{VideoConfiguration/contentType}}.
+          </li>
+          <li>
+            If |mimeType| is <code>failure</code>, return <code>false</code> and
             abort these steps.
           </li>
+          <li>
```

Comment:
If you keep the original order of the steps, I think it makes sense to set the slot value after the steps below are complete, i.e. there are no other reasons to reject the `VideoConfiguration`.